Doist/reactist
# Refactor build system to support CSS modules

## Short description

This builds infrastructure for a single rollup build that supports:

* CSS modules
* Code splitting just [like how antd does it](https://unpkg.com/browse/antd@4.3.0/).
* Conversion of SVG to inline data urls.

This also removes the previous webpack builder as that is no longer necessary with this workflow.

## Test Plan
This adds infrastructure to support CSS modules. Nothing has yet been made CSS module yet, as that needs additional changes in Twist & Todoist, but the builder has been completely changed. To test:

1. Checkout reactist
2. npm run build
3. Checkout twist-web
4. npm i; npm run reactist:link; npm run start
5. Make sure everything works fine.
6. Do same steps for todoist-web

Note there are 2 PRs, [one Twist](https://github.com/Doist/twist-web/pull/2497), and [one Todoist](https://github.com/Doist/todoist-web/pull/1678) that have to be reviewed with this together.
 
## PR Checklist
-   <strike>[ ] Added tests for bugs / new features</strike>
-   [x] Updated docs (storybooks, readme)
-   [x] Described changes in `CHANGELOG.md`
-   [x] Executed `npm run lint` and made sure no errors / warnings were shown
-   [x] Executed `npm run test` and made sure all tests are passing
-   [x] Bumped version in `package.json`
-   [x] Updated all static build artifacts (`npm run build-all`)
